colossalai/nn/loss/loss_moe.py

```diff
@@ -1,7 +1,7 @@
 import torch.nn as nn
 from colossalai.registry import LOSSES
 from torch.nn.modules.loss import _Loss
-from colossalai.global_variables import moe_env
+from colossalai.core import moe_context as moe_env

```

Comment:
I let you change the name of global variable in previous PR
Use upper case and GLOBAL_ as a prefix.

Reply:
used MOE_CONTEXT instead